### PR TITLE
Chore, updates operator-sdk, check todos

### PR DESCRIPTION
**Pull request by portly-halicore-76**
Updates opr-sdk to comply with latest version upstream.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,20 +9,16 @@ ARG GONOSUMDB="proxy.golang.org/*,github.com,github.com/*"
...
-ARG cert_location=/usr/local/share/ca-certificates
-
 WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
 
 # Copy the go source
-COPY main.go main.go
+COPY cmd/main.go cmd/main.go
```
Comment (devotional-phoenix-97) [inline Dockerfile]: operator-sdk (I guess this is the old operator convention) has this in /main.go
Seems like (as you emntioned) new convention is /cmd/main.go, but this is not inline with standard golang convention which would be /cmd/[folder]/main.go (where [folder] is a seperate folder for every binary to be built from this codebase).
CNPG seems to follow the GoLang convention (as they have /cmd/manager/main.go, and /cmd/kubectl-cnpg/main.go.
Let's use that convention and also move the cli stuff there too.
-  /cmd/manager/main.go (formerly /main.go)
-  /cmd/crypttool/main.go (formerly /cli/crypttool/main.go)
-  /cmd/webservice/main.go (formerly /cli/webservice/main.go, which will probably get its own repo).

**Review by hikarukin — DISMISSED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -9,20 +9,16 @@ ARG GONOSUMDB="proxy.golang.org/*,github.com,github.com/*"
...
 COPY api/ api/
 COPY cli/ cli/
-COPY controllers/ controllers/
```
Comment (devotional-phoenix-97) [inline Dockerfile:23]: Deze code hebben we toch in de docker container nodig?

**Review by devotional-phoenix-97 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (portly-halicore-76) [reply Dockerfile]: cmd/main.go is juist wat operator-sdk nu doet.

Comment (portly-halicore-76) [reply Dockerfile]: Done, eends

**Review by hikarukin — APPROVED** [review]

**Review by hikarukin — APPROVED** [review]

**Pull request merged**